includes/class-wc-siftscience-admin.php

```diff
@@ -292,6 +303,16 @@ private function get_section_fields( $sub_section ) {
 						)
 					),
 
+					$this->wc_element->create(
+						WC_SiftScience_Element::CUSTOM,
+						'Cid',
+						'disc label',
+						implode( '|', $this->status->get_status_options() ),
+						array(
+							'callback' => 'gb_callback',
```

Comment:
Instead of using implode/explode, you can just place the array as a new entry here.

Reply:
?